scripts/incentives-setup.sh

```diff
@@ -0,0 +1,87 @@
+#!/bin/bash -ex
+
+
+cleanup() {
+    kill $dualityd_pid >/dev/null 2>&1
+}
+trap cleanup ERR
+trap 'exit_code=$?; cleanup; exit $exit_code' EXIT
+
+rm -rf ~/.duality
+dualityd init local-node
+dualityd add-consumer-section
+
+epochs_genesis='{
+    "epochs": [
+        {
+            "identifier": "day",
+            "start_time": "0001-01-01T00:00:00Z",
+            "duration": "10s",
+            "current_epoch": "0",
+            "current_epoch_start_time": "0001-01-01T00:00:00Z",
+            "epoch_counting_started": false,
+            "current_epoch_start_height": "0"
+        },
+        {
+            "identifier": "hour",
+            "start_time": "0001-01-01T00:00:00Z",
+            "duration": "3600s",
+            "current_epoch": "0",
+            "current_epoch_start_time": "0001-01-01T00:00:00Z",
+            "epoch_counting_started": false,
+            "current_epoch_start_height": "0"
+        },
+        {
+            "identifier": "week",
+            "start_time": "0001-01-01T00:00:00Z",
+            "duration": "604800s",
+            "current_epoch": "0",
+            "current_epoch_start_time": "0001-01-01T00:00:00Z",
+            "epoch_counting_started": false,
+            "current_epoch_start_height": "0"
+        }
+    ]
+}'
+dasel put -f ~/.duality/config/genesis.json -t json -v "$epochs_genesis" '.app_state.epochs' 
```

Comment:
I don't think it's worth the time right now to get the dockerfile updated to support dasel. I can remove this file if you think it's necessary but seems like it could be useful in the future.

Reply:
I think this file is still a useful example 👍